scripts/shared/nodeinfo/nodeinfo-update.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+cp /opt/tomesh/nodeinfo.json /tmp
+
+# Replace placeholders with dynamic info
+sed -i -e "s/__KEY__/$(grep -m 1 '"ipv6"' /etc/cjdroute.conf | awk '{ print $2 }' | sed 's/[",]//g')/g" /tmp/nodeinfo.json
```

Comment:
Why isn't __KEY__ in the install file?

Reply:
Because CJDNS key CAN change (unlikely but can) and we can easy extrapolate it on boot so why not.

Git data that is part of the install is a little harder since the git repo can be deleted after the install, and we don't know WHERE it was downloaded to.